UnitTests/Tests/Formatters/TRTemperatureComparisonFormatterSpec.m

```diff
@@ -0,0 +1,34 @@
+#import "UnitTests-Swift.h"
+#import "TRTemperatureComparisonFormatter.h"
+
+@interface TRTemperatureComparisonFormatter (Tests)
+
++ (NSCalendar *)calendar;
+
+@end
+
+SpecBegin(TRTemperatureComparisonFormatter)
+
+NSDate* (^dateFromString) (NSString*) = ^NSDate* (NSString *dateString) {
```

Comment:
Can you move these pointers to the other side of the space? (`NSDate *(` instead of `NSDate* (`)